smallab/checkpointed_experiment_handler.py

```diff
@@ -84,7 +92,7 @@ def __save_checkpoint(self, experiment, name, specification):
         try:
             location = get_partial_save_directory(name, specification)
             os.makedirs(location, exist_ok=True)
-            #TODO make sure a checkpoint with this name doesn't already exist
+            # TODO make sure a checkpoint with this name doesn't already exist
```

Comment:
What will Smallab do if there is a checkpoint naming conflict?

Reply:
override that file instead of making a new file. it's not catastrophic its just possible the program would crash while re-writing that file causing you to have to go back to the older backup (that's part of the reason i chose 3 backups)